requirements-http-client.txt

```
@@ -1,3 +1,3 @@
 click==8.1.7
-httpx==0.27.0
-rich==13.7.1
+httpx==0.27.2
```

```suggestion
httpx==0.27.0
```

Lets try downgrading httpx